service-base/src/main/java/org/eclipse/hono/service/registration/BaseRegistrationService.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2016, 2017 Bosch Software Innovations GmbH.
+ * Copyright (c) 2016, 2018 Bosch Software Innovations GmbH.
```

Comment:
skipped? WDYM?

Reply:
I thought: 
if there would not have been changes in 2017, only in 2016 and 2018, then I would expect this.
Otherwise either 2016,2017,2018 or 2016-2018.
WDYT?